src/audio/copier/copier.c

```diff
@@ -1223,11 +1223,9 @@ static int do_endpoint_copy(struct comp_dev *dev)
 
 		return ret;
 	} else {
-		if (dev->ipc_config.type == SOF_COMP_HOST && !cd->ipc_gtw) {
-			if (dev->state != COMP_STATE_ACTIVE)
-				return 0;
```

Comment:
these two lines were added in an earlier patch in this PR. Were they needed there? If not, better not to add them at all. If you're removing redundant code from the current version, would be good to have that as a separate patch in the PR

Reply:
let me remove from beginning, it should not be needed.